charts/orchestrator/values.yaml

```diff
@@ -119,4 +121,3 @@ orchestrator:
   sonataflows: # workflows to get deployed - this option will be removed once the plugin will interact directly with the data-index
     - name: greeting
       image: quay.io/masayag/serverless-workflow-greeting:latest
```

Comment:
Should we move this image to https://quay.io/organization/orchestrator as well?

Reply:
this we can do, I'll create a separate issue to track this.

Reply:
thank you